bloom/commands/release.py

```diff
@@ -707,6 +749,24 @@ def get_github_interface(quiet=False):
     return gh
 
 
+def get_gitlab_interface(server, quiet=False):
+    global _gl
+    if _gl is not None:
+        return _gl
+    # Make sure we can import gitlab
+    try:
+        import gitlab
```

Comment:
I think you can safely import this up top if you add it to https://github.com/locusrobotics/bloom/blob/locus-master/setup.py#L6, that should ensure dev machines pull the library down as well.